vignettes/wip/demo.Rmd

```diff
@@ -1,32 +1,54 @@
 ---
-title: "demo"
+title: "Using Fledge"
 output: rmarkdown::html_vignette
 vignette: >
   %\VignetteIndexEntry{demo}
   %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
+In this guide, we will illustrate how to use `fledge` functionality with a dummy R package.
+
 ```{r setup}
 options(crayon.enabled = TRUE)
 fansi::set_knit_hooks(knitr::knit_hooks)
 ```
 
+## Setting up the development environment
+
+Let us say you have a great new idea that you wish to implement and eventually release to CRAN. 
 
-Create new package
+You would start by creating a new package. We use the `tempfile()` and `create_package()` functions for this. 
+
+The output shows the details of the package created. 
 
 ```{r }
 pkg <- usethis::create_package(tempfile("pkg"))
+```
+
+You can check the location of the new package as follows
+
+```{r }
 pkg
 ```
 
-Set up development work
+Next activity on the list would be to set up this package for development and create a git repository for the same. Following would be the sequence of steps you would most likely follow.
```

Comment:
The code in this chunk is necessary only for the demonstration here, in a realistic scenario everything will be set up already. Don't need to elaborate further here.